cruise.umple/src/Generator_CodeGvClassDiagram.ump

```diff
@@ -129,6 +129,9 @@ class GvClassDiagramGenerator
     if (hasSuboption("showmethods")) {
       boolean isFirstMethod = true;
       for (Method uMethod : inUmpleClassifier.getMethods()) {
+      	if (uMethod.getSource() == Method.Source.fAutoAPI)
+      	  continue;
+
```

Comment:
I wonder why you wanted to escape these methods here?
Actually, it's a nice way to show what them in the model. By default, we don't shows any methods, but when they are selected they should be shown